src/scripting/game_lua_kernel.cpp

```diff
@@ -103,6 +103,7 @@
 #include "units/map.hpp"  // for unit_map, etc
 #include "units/ptr.hpp"                 // for unit_const_ptr, unit_ptr
 #include "units/types.hpp"    // for unit_type_data, unit_types, etc
+#include "formula/string_utils.hpp"    // for unit_type_data, unit_types, etc
```

Comment:
The only reason to comment and include is if a code-reader would go "Wha? Why's that here?" usually because it's needed for one, easily overlooked constant or something like that. Otherwise, there's no need to comment an include and the code-reader will properly think, "This will be used throughout."